sdk/state/payment_test.go

```diff
@@ -252,13 +287,16 @@ func TestChannel_ConfirmPayment_initiatorRejectsPaymentToRemote(t *testing.T) {
 		IterationNumber: 2,
 		Balance:         110, // Local (initiator) owes remote (responder) 110, payment of 10 from ❌ local to remote.
 	}
```

Comment:
nit: can we change the name of this test to `localRejectsPaymentToRemote_initiator`? since initiator could be remote, just noticed this now when trying to remember what this test is doing

and then below `localRejectsPaymentToRemote_responder`

Reply:
Good call. 3ac0290f75148e13c2392c8f891bbeccb92a783f